LAMP/DeleteContact.php

```diff
@@ -0,0 +1,39 @@
+<?php
+	$inData = getRequestInfo();
+
+	$color = $inData["color"];
+	$userId = $inData["userId"];
+
+	$conn = new mysqli("localhost", "TheBeast", "WeLoveCOP4331", "COP4331");
+	if ($conn->connect_error)
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		$stmt = $conn->prepare("DELETE from Contacts where UserID =?");
```

Comment:
Yes we want to make sure we are deleting from the right User BUT This statement is saying Delete Any Contacts that is associated with this UserID. What we really want to check for is `WHERE UserID=? AND ContactID=?`